x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/DerivDoubleAggregator.java

```diff
@@ -24,15 +24,15 @@
     {
         @IntermediateState(name = "count", type = "LONG"),
         @IntermediateState(name = "sumVal", type = "DOUBLE"),
-        @IntermediateState(name = "sumTs", type = "LONG"),
+        @IntermediateState(name = "sumTs", type = "DOUBLE"),
         @IntermediateState(name = "sumTsVal", type = "DOUBLE"),
-        @IntermediateState(name = "sumTsSq", type = "LONG") }
+        @IntermediateState(name = "sumTsSq", type = "DOUBLE"), }
```

Comment:
This change breaks BWC tests in serverless. Can you make sure to guard the deriv tests with the new capacity?